yew/Cargo.toml

```diff
@@ -142,6 +142,8 @@ agent = ["bincode"]
 yaml = ["serde_yaml"]
 msgpack = ["rmp-serde"]
 cbor = ["serde_cbor"]
+listener_tests = []
+listener_benchmarks = []
```

Comment:
do we really need these extra features?

Reply:
I need some way to run listener tests and benchmarks separately from all other tests. This is because they are currently the only tests that mount an `App` and that seems to conflict with other tests. If you know a way to do that without features, do tell.

Reply:
Hmm, good point. I'll see if I can find a way around that because I'd really like to keep the feature count down a bit.